src/devices/proformtreadmill/proformtreadmill.cpp

```diff
@@ -378,7 +378,7 @@ void proformtreadmill::update() {
                 writeCharacteristic(noOpData4, sizeof(noOpData4), QStringLiteral("noOp"), true);
                 break;
             case 4:
-                writeCharacteristic(noOpData2, sizeof(noOpData2), QStringLiteral("noOp"));
```

Comment:
@Novamusix why change this if you created a new profile for your treadmill?

Reply:
You are correct. This was a left over from my initial investigation before splitting the profile. I noticed that noOpData5 was not being referenced where noOpData2 was referenced in both case 2 and 4. Unlike other profiles such as norditrack_s25i which referenced all the noOpData fields in order. This change can be ignored.